ortools/sat/csharp/CpSolver.cs

```diff
@@ -11,12 +11,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-using System;
-using System.Collections.Generic;
 using System.Runtime.CompilerServices;
+using Google.OrTools.Sat;
 
-namespace Google.OrTools.Sat
```

Comment:
Why did you remove the namespace ?
i.e. `CpSolver` should still be inside the `Google.OrTools.Sat` IMHO (and the new `using` directive won't make it)